documentation/introduction.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: Introduction
+---
+
+luent Ribbon Control Suite is a library that implements Microsoft® Office Fluent™ user interface for the Windows Presentation Foundation (WPF).
+
+### Ribbon control
+
+What is a ribbon? A ribbon is a control which consists of several tabs, grouped by functionality to represent the menu actions of a software programme.
+
+![Ribbon overview](http://download-codeplex.sec.s-msft.com/Download?ProjectName=fluent&DownloadId=102041)
```

Comment:
Please don't reference resources from CodePlex.
If you need images you should add them to this repository.